src/components/Navigation.js

```diff
@@ -14,8 +16,20 @@ import { LOGO_URL, SMALL_LOGO_URL } from '../constants'
 import { capitalizeWord, matchTitlePage } from '../scripts/selectors'
 import { pageName } from '../reactivities/variables'
 
+const useStyles = makeStyles((theme) => ({
```

Comment:
There's a strong reason why you placed the styling up here and not in the `Navigation.scss` file? I don't see this pattern used anywhere else on the app

Reply:
The main reason was to make the header responsive. For some reason in the scss file, it wasn't working properly

Reply:
try moving this into the scss file using the `height` property and the `@media screen` to make it responsive, we should try to maintain styling in a separate file. @ at me on slack if you keep having problems with this